### PR TITLE
chore(containers): remove warnings in tests

### PR DESCRIPTION
**Pull request by jmfrancois**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

Lots of warnings during the tests.

**What is the new behavior?**

No warnings at all:

screen shot 2017-03-16 at 17 56 32

**Does this PR introduce a breaking change?**

- [x] No

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://292.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://292.talend.surge.sh)

File: packages/containers/src/List/List.container.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ class List extends React.Component {
 			sort: PropTypes.object,
 			filter: PropTypes.object,
 			pagination: PropTypes.shape({
-				onChange: PropTypes.func,
+				onChange: PropTypes.string,
```
Comment (frassinier) [inline packages/containers/src/List/List.container.js]: Hm?

Comment (jmfrancois) [inline packages/containers/src/List/List.container.js]: answer is here https://github.com/Talend/ui/blob/master/packages/containers/src/List/List.container.js#L168

**Pull request merged**